src/runtime/node/perf_hooks/internal/constants.ts

```diff
@@ -1,15 +1,154 @@
 import type perf_hooks from "node:perf_hooks";
 
-export const constants: typeof perf_hooks.constants = {
-  NODE_PERFORMANCE_GC_MAJOR: 4,
-  NODE_PERFORMANCE_GC_MINOR: 1,
-  NODE_PERFORMANCE_GC_INCREMENTAL: 8,
-  NODE_PERFORMANCE_GC_WEAKCB: 16,
-  NODE_PERFORMANCE_GC_FLAGS_NO: 0,
-  NODE_PERFORMANCE_GC_FLAGS_CONSTRUCT_RETAINED: 2,
-  NODE_PERFORMANCE_GC_FLAGS_FORCED: 4,
-  NODE_PERFORMANCE_GC_FLAGS_SYNCHRONOUS_PHANTOM_PROCESSING: 8,
-  NODE_PERFORMANCE_GC_FLAGS_ALL_AVAILABLE_GARBAGE: 16,
-  NODE_PERFORMANCE_GC_FLAGS_ALL_EXTERNAL_MEMORY: 32,
-  NODE_PERFORMANCE_GC_FLAGS_SCHEDULE_IDLE: 64,
-};
+export const constants = Object.create(
+  null,
+) satisfies typeof perf_hooks.constants;
+
+// captured from Node.js v22.3.0 using
+// Object.getOwnPropertyDescriptors(require('perf_hooks').constants)
+Object.defineProperties(constants, {
+  NODE_PERFORMANCE_GC_MAJOR: {
+    value: 4,
+    writable: false,
+    enumerable: true,
+    configurable: false,
+  },
```

Comment:
Wondering if an `Object.freeze()` would result in almost identical behavior to reduce size

Reply:
many of the values are non-enumerable and that's why I went with `defineProperties`, but you are right, this can be written in a more size-efficient way. I'll redo.

Reply:
how about this 29720492bf1ee4ddfaee7eb1e1e438297da398cc - I think this is as small as it gets while staying API surface compliant with Node.